src/main/java/mate/academy/ConnectionUtil.java

```diff
@@ -0,0 +1,27 @@
+package mate.academy;

```

Comment:
```suggestion
package mate.academy.util;

```
better move to util package